base/gfspclient/authorizer.go

```diff
@@ -32,3 +33,52 @@ func (s *GfSpClient) VerifyAuthorize(ctx context.Context,
 	}
 	return resp.GetAllowed(), nil
 }
+
+// GetAuthNonce get the auth nonce for which the Dapp or client can generate EDDSA key pairs.
+func (s *GfSpClient) GetAuthNonce(ctx context.Context, in *gfspserver.GetAuthNonceRequest, opts ...grpc.CallOption) (*gfspserver.GetAuthNonceResponse, error) {
```

Comment:
GfSpClient should seem like SDK, the GfSpClient will be offer to other repo, it should not use the gfspserver.GetAuthNonceRequest(Grpc specs), may be gfspapp support other protocol(http/ other rpc), so the client params should use the meaningful params, eg:  accountId, domain.

Reply:
Done